hal_st/stm32fxxx/GpioStm.cpp

```diff
@@ -558,7 +562,12 @@ namespace hal
                 EXTI->PR &= (1 << line); // Interrupt pending is cleared by writing a 1 to it
 #endif
                 if (handlers[line])
-                    infra::EventDispatcher::Instance().Schedule(handlers[line]);
+                {
+                    if (interruptTypes[line] == InterruptType::dispatched)
+                        infra::EventDispatcher::Instance().Schedule(handlers[line]);
```

Comment:
With ExtiHandler now being an immediate interrupt handler, quick toggles of a GPIO line can easily blow up the event dispatcher